leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -165,6 +168,7 @@ private void proposeLeadershipOrWaitForBackoff(LeadershipState currentState)
         }
 
         long backoffTime = (long) (randomWaitBeforeProposingLeadership * Math.random());
+        log.debug("Waited for [{}] ms before proposing leadership", SafeArg.of("waitTimeMs", backoffTime));
```

Comment:
The log message indicates that the wait has already occurred, but the sleep is below. Seems like we either want to move the log line after sleep or adjust message wording indicating that it will wait for given backoff